CHANGELOG.rst

```diff
@@ -7,6 +7,17 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.1 (2024-11-27)
+--------------------------
+
+* Fixed: ensure we always use MD5 sums
+  when comparing files
+* Removed: ``audbackend.checksum()``,
```

Comment:
I would rephrase this to something like

"Removed audbackend.checksum() because the special handling of Parquet file metadata checksums introduced potential issues or complications."

Reason: For me the "introduced wrongly" sounds a bit weird - I cannot motivate whether it is incorrect though.

Reply:
Thanks for the suggestion, I was also not very happy with my text. I updated the text accordingly.